### PR TITLE
post in channel when user completes an action

### PR DESCRIPTION
**Pull request by chance-cyphers**
I would love it if you know some nicer way to pipe the action item description down to where it's needed. Not too happy about how it is, but it doesn't seem important enough to care too much about right now.

### CODE REVIEW

File: src/handlers/block_action.handler.ts [hunk 1]
```diff
@@ -131,11 +131,18 @@ function handleCompleteAction(payload: any, response: express.Response, actionId
...
     const queryValues = [actionId];
 
-    client.query(queryText, queryValues)
+    let itemDescription = "an item";
+
+    client.query('select * from action_items where id=$1', [actionId])
+        .then(res => {
+            itemDescription = res.rows[0].description;
+            return client.query(queryText, queryValues);
+        })
```
Comment (tchupp) [inline src/handlers/block_action.handler.ts:145]: If we used `async/await` here instead of `.then`, it might look a little nicer

Comment (chance-cyphers) [reply src/handlers/block_action.handler.ts:145]: good idea. Lemme try that out...

Comment (chance-cyphers) [reply src/handlers/block_action.handler.ts]: looks better to me 🤷‍♂️

File: src/handlers/block_action.handler.ts [hunk 2]
```diff
@@ -38,7 +38,10 @@ function routeBlockActions(payload: any, response: express.Response, client: Cli
         handlePost(payload, response, client);
     } else if (actionId.includes("complete")) {
         const docId = actionId.split(":")[1];
-        handleCompleteAction(payload, response, docId, client);
+        handleCompleteAction(payload, response, docId, client)
+            .catch(err => {
+                console.error(`error in complete flow: ${err}`);
+            });
```
Comment (tchupp) [inline src/handlers/block_action.handler.ts:44]: I like moving this out here!

**Pull request merged**